extensions/markdown-language-features/src/features/previewManager.ts

```diff
@@ -220,6 +224,11 @@ export class MarkdownPreviewManager extends Disposable implements vscode.Webview
 			this._staticPreviews.delete(preview);
 		});
 
+		// Continuously update the scroll info in case user changes the editor.
+		preview.onScroll((scrollInfo) => {
+			this._topmostLineMonitor.previousStaticEditorInfo = scrollInfo;
```

Comment:
Should this live in the preview itself?

Reply:
Yup, accidentally forgot that I passed an instance of the topmost line monitor into the preview. Preview should now do this.